src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -306,12 +307,23 @@ private async Task<PullRequest> SubmitPRAsync(string packageId, string version,
             var upstreamMasterSha = upstreamMaster.Object.Sha;
 
             Reference newBranch = null;
+
             try
             {
                 var retryPolicy = Policy.Handle<ApiException>().WaitAndRetryAsync(3, i => TimeSpan.FromSeconds(i));
                 await retryPolicy.ExecuteAsync(async () =>
                 {
-                    await this.github.Git.Reference.Create(repo.Id, new NewReference($"refs/{newBranchNameHeads}", upstreamMasterSha));
+                    try
+                    {
+                        Reference reference = await this.github.Git.Reference.Create(repo.Id, new NewReference($"refs/{newBranchNameHeads}", upstreamMasterSha));

```

Comment:
>Reference reference [](http://example.com/codeflow?start=24&length=19)

seems like this local variable is assigned but not used?

Reply:
Removed unused local variable